### PR TITLE
Package guardian.0.1.0

### PR DESCRIPTION
**Pull request by mabiede**
### `guardian.0.1.0`
Role-based access control for OCaml
Defined actors can access targets according to specified rules (allowed actions: CRUD).

---
* Homepage: https://github.com/uzh/guardian
* Source repo: git+https://github.com/uzh/guardian.git
* Bug tracker: https://github.com/uzh/guardian/issues

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

File: packages/guardian/guardian.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  "caqti-driver-mariadb" {>= "1.8.0"}
+  "caqti-lwt" {>= "1.8.0"}
+  "containers" {>= "3.6"}
+  "dune" {>= "2.9" & >= "2.9.0"}
+  "logs" {>= "0.7.0"}
+  "lwt" {>= "5.6.1"}
+  "lwt_ppx" {>= "2.1.0"}
+  "mariadb" {>= "1.1.4"}
+  "ocaml" {>= "4.12.0"}
+  "ocamlformat" {>= "0.24.1"}
```
Comment (kit-ty-kate) [inline packages/guardian/guardian.0.1.0/opam:24]: I can't find anything in the code that uses ocamlformat. Is this just a dependency that you use personally for development?

File: packages/guardian/guardian.0.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+]
+license: "GPL-2.0-or-later"
+homepage: "https://github.com/uzh/guardian"
+doc: "https://uzh.github.io/guardian"
+bug-reports: "https://github.com/uzh/guardian/issues"
+depends: [
+  "caqti-driver-mariadb" {>= "1.8.0"}
+  "caqti-lwt" {>= "1.8.0"}
+  "containers" {>= "3.6"}
+  "dune" {>= "2.9" & >= "2.9.0"}
```
[the remark this answers is not in the document]
Comment (mseri) [reply packages/guardian/guardian.0.1.0/opam:18]: To fix this altogether, this entry in the dune-project should be removed: https://github.com/uzh/guardian/blob/16b6b00a684ce9e6bba8b85a5a28465b4cf65f8a/dune-project#L38

**Comment by mabiede** [comment]
@kit-ty-kate @mseri thanks for spotting and reporting those dependencies. I could remove those.

**Comment by mabiede** [comment]
I've found a type signature bug. I'll clean up this as well

**Comment by mseri** [comment]
Looks good now, thanks

**Pull request merged**